### DISCUSSION TITLE
Catalina 5600g

### DISCUSSION THREAD

**Discussion by MatitaPere** [ChefKissInc/NootedRed#155] [Q&A] [answered]
Just wondering if anyone got nRed working with Catalina and 5600g. Thanks

**Comment by VisualEhrmanntraut** [comment, marked as answer]
It should work now

**Reply by luckiest-luk** [reply]
My CPU is R5 5600H, and using the latest kext, it still displays a black screen. I extracted these from the system log to see if it helps. thank you for your contribution!
[IoRegistry.log](https://github.com/ChefKissInc/NootedRed/files/12575806/IoRegistry.log)
[panic.log](https://github.com/ChefKissInc/NootedRed/files/12575807/panic.log)
[panic1.log](https://github.com/ChefKissInc/NootedRed/files/12575808/panic1.log)
[panic2.log](https://github.com/ChefKissInc/NootedRed/files/12575809/panic2.log)

**Reply by VisualEhrmanntraut** [reply]
You are experiencing a similar issue to @Zormeister
